WIP-SQL/what-is-a-database-server.md

```diff
@@ -0,0 +1,61 @@
+# What is a database server?
+author: SebaRaba
+
+levels:
+
+  - beginner
+
+  - basic
+
+type: normal
+
+category: must-know
+
+links:
+
+  - '[More on database servers](http://ecomputernotes.com/fundamental/what-is-a-database/what-is-a-database-server)'
+
+---
+## Content
+
+A database server can be the *hardware* part of a database. A computer dedicated to perform database storage an retrieval. Such a hardware runs the database software  application (DBMS - database management system). It receives requests from the **client machines** and searches the collection for the **requested records**.
+
+As a software, it means the backend (also called instance) part of an database application. The backend handles tasks such as data analisys, storage, data manipulation and other non-user specific tasks. In general, database management systems provide database-server functionality. There are some DBMS[1] (such as MySQL) which rely exclusively on the *client-server model*[2].
+
+
+Note it's not mandatory that the actual data is hosted *on the same hardware server* as the database application.
```

Comment:
I think that you're trying to talk about sharding and replication here, but without going into it this is kind of misleading. Probably strike this.

Say something like, "A group of database servers (called a **Cluster**) can coordinate to share data across multiple physical pieces of hardware. This is used for very large databases, or for redundancy. For this reason, the term Database Server increasingly refers to the software, instead of the hardware."